docs/pages/dsDisplay/ds-display_L3_Test-Procedure.md

```diff
@@ -0,0 +1,305 @@
+# Device Settings Display L3 Test Case and Procedure Documentation
+
+## Table of Contents
+
+- [Overview](#overview)
+- [Acronyms, Terms and Abbreviations](#acronyms-terms-and-abbreviations)
+- [References](#references)
+- [Setting Up Test Environment](#setting-up-test-environment)
+- [Test Case Procedure](#test-case-procedure)
+  - [dsDisplay_test01_VerifyDisplayConnectCallBackTest.py](#dsdisplay_test01_verifydisplayconnectCallBackTestpy)
+  - [dsDisplay_test02_TestVerifyDisplayEdid.py](#dsdisplay_test02_testverifydisplayEdidpy)
+  - [dsDisplay_test03_AspectRatioVerificationTest.py](#dsdisplay_test03_aspectratioverificationtestpy)
+  - [dsDisplay_test04_TestVerifyDisplayEdidBytes.py](#dsdisplay_test04_testverifydisplayEdidbytespy)
+
+## Overview
+
+This document describes the L3 Test case Procedure Documentation for the Device Settings Display module.
```

Comment:
Rephrase this.  probably "This document describes the VTS Level 3 Test scenarios and the execution process for the Device Setting Display Module"

Reply:
Done